### ISSUE TITLE
One possible mistake in C18 problem 18.1-3

### ISSUE THREAD

**Issue by tsato-code** [gzc/CLRS#230]
I think that the answer is a bit short.

Text says "Every node may contain at most 2t - 1 keys".
If t = 2  then the B-tree root note contain at most 3 keys.
As a result, the following is also the answer.

 2 4
1 3 5

Don't think so?

**Comment by samolisov** [comment]
I agree with your answer and have proposed a PR #295. If we have a look at fig 18.1, we have a chance to see something a bit similar: 

```
      D, H
B,C F,G  J,K,L
```
It can be a child node between keys in the parent.
